nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -404,6 +405,13 @@ int ddxProcessArgument(int argc, char *argv[], int i)
     return 1;
   }
 
+  if (!strcmp(argv[i], "-nxrealwindow")) {
+    if (++i < argc) {
```

Comment:
`-ac :66 -nxrealwindow` will crash

You introduced the same kind of bug that I have fixed in 19116a61c98684fc5c0256b65f49462457bb0f5b Please check for the fix there.